examples/server-customobject.py

```diff
@@ -16,28 +22,44 @@
     uri = "http://examples.freeopcua.github.io"
     idx = server.register_namespace(uri)
 
-    # Import customobject type
-    server.import_xml('customobject.xml')
-
     # get Objects node, this is where we should put our custom stuff
     objects = server.get_objects_node()
 
+    # Example 1 - create a basic object
+    #-------------------------------------------------------------------------------
+    myobj = objects.create_object(idx, "MyObject",)    
+    #-------------------------------------------------------------------------------
+
+
+    # Example 2 - create a new object type and a instance of the new object type
+    #-------------------------------------------------------------------------------
+    types = server.get_node(ua.ObjectIds.BaseObjectType)
```

Comment:
You mean line squash 36+37 ?
Line 37 did contain an error create_object_type should be create_object_type. Already fixed and committed:
```python
    mycustomobj_type = types.create_object_type(idx, "MyCustomObject")
```
I think about replacing line 36 it by the following construction:
```python
    # types = server.get_node(ua.ObjectIds.BaseObjectType) # this only a shortcut for standard types
    object_type_to_derive_from = server.get_root_node().get_child(["0:Types", 
                                                                   "0:ObjectTypes", 
                                                                   "0:BaseObjectType"])
```
This more descriptive, makes it more clear how to extended the code to derive from even your own types.